src/admin/messages/messages.scss

```diff
@@ -13,3 +13,7 @@
 .badge-info {
   background-color: #3a87ad;
 }
+
+.messages-textField {
+  max-width: 150px;
```

Comment:
@0nat Could you use variables from [here](https://github.com/motech/motech-ui/blob/master/src/app/app.variables.scss)?